velox/functions/prestosql/tests/utils/FunctionBaseTest.h

```diff
@@ -106,6 +106,23 @@ class FunctionBaseTest : public testing::Test,
     return evaluate(typedExpr, data, rows);
   }
 
+  VectorPtr evaluateWithCustomQueryConfig(
```

Comment:
You can just set this in your test case using : 

``` 
queryCtx_->testingOverrideConfigUnsafe({ .. })
``` 

instead of creating this.